a10_octavia/common/a10constants.py

```diff
@@ -101,3 +101,4 @@
 HANDLE_VRID_MEMBER_SUBFLOW = 'handle-vrid-member-subflow'
 SPARE_VTHUNDER_CREATE = 'spare-vthunder-create'
 LB_TO_VTHUNDER_SUBFLOW = 'lb-to-vthunder-subflow'
+NONE_STR = ["None"]
```

Comment:
There isn't a real need to make a constant for this. It isn't widely used nor is it continually subject to change like flow names are.

Reply:
removed it.